lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -100,6 +101,16 @@ namespace mx = MaterialX;
 
 PXR_NAMESPACE_OPEN_SCOPE
 
+TF_DEFINE_ENV_SETTING(
+    MAYAUSD_DISABLE_ASYNC_TEXTURE_LOADING,
+    0,
+    "Disable asynchronous texture loading.");
+static bool _IsDisabledAsyncTextureLoading()
+{
+    static bool _v = TfGetEnvSetting(MAYAUSD_DISABLE_ASYNC_TEXTURE_LOADING) == 1;
```

Comment:
We'll need a runtime switch to turn on and off the async loading so that test automation can control what is going on. Something that can be set through python would be good.

Reply:
I changed the env var to be an Maya `optoinVar` called **mayaUsd_DisableAsyncTextureLoading** :
https://github.com/Autodesk/maya-usd/pull/1980/files#diff-79f656a2d9ef20b627475bb4041dad2cd7d14cf53ec94fafb7a18669fe228a6bR66-R69

Let me know if this is breaking any `maya-usd` development guideline?